draft-ietf-scitt-architecture.md

```diff
@@ -274,7 +260,7 @@ Transparency Services use `sub` to identify the entity about which they are issu
 Transparency Service:
 
 : an entity that maintains and extends the Append-only Log, and endorses its state.
-A Transparency Service MAY implement a Registration Policy, often referred to by its synonym Notary.
+A Transparency Service MAY alter the contents of the unprotected header of Signed Statements, this will alter the binary content stored on the log, from which receipts are produced.
```

Comment:
Does it mean that unprotected headers are now also included in the receipt's signed information? This significantly complicates the interpretation of the resulting receipts by verifiers. I also don't think it should be discussed in the terminology.

Reply:
They have always been included, this is just making that explicit.

Reply:
Editors meeting note: please open a PR/Issue if needed.